src/afl-showmap.c

```diff
@@ -1339,6 +1333,8 @@ int main(int argc, char **argv_orig, char **envp) {
 
   }
 
+  if (collect_coverage) { binary_mode = false; }  // ensure this
```

Comment:
Should we assert and FATAL if it's set, instead?

Reply:
I don’t care, should never happen :)